photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextureRenderer.java

```diff
@@ -65,7 +65,18 @@
 
     private static final int FLOAT_SIZE_BYTES = 4;
 
+    private boolean mIsInitialized = false;
```

Comment:
Is this something that could be sent as a separate PR to the origin?

Reply:
Yes, it's a fix. Maybe can be upstreamed.